hack/windows/Helper.psm1

```diff
@@ -93,9 +94,12 @@ function Install-AntreaAgent {
 
     New-DirectoryIfNotExist $KubernetesHome
     # Download kubectl
-    Get-WebFileIfNotExist $kubectl  "https://$KubernetesURL/$KubernetesVersion/bin/windows/amd64/kubectl.exe"
+    cd "$KubernetesHome"
+    curl.exe -LO "https://$KubernetesURL/$KubernetesVersion/bin/windows/amd64/kubectl.exe"
+    # Get-WebFileIfNotExist $kubectl "https://$KubernetesURL/$KubernetesVersion/bin/windows/amd64/kubectl.exe"
```

Comment:
Why comment out the command `Get-WebFileIfNotExist`? I thought you wanted to use it.

Reply:
This is for testing. Has changed it back.